packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -460,12 +467,17 @@ export default function() {
               }
               path.replaceWithMultiple(nodes);
             } else if (path.isExportAllDeclaration()) {
-              const exportNode = buildExportAll({
+              const obj = {
                 OBJECT: addRequire(
                   path.node.source.value,
                   path.node._blockHoist,
                 ),
-              });
+              };
+
+              let exportNode = buildExportAll(obj);
```

Comment:
Nit, can avoid calling if `this.opts.loose` `buildExportAll` via ternary?

Reply:
`const exportNode = (this.opts.loose)? buildExportAllLoose(obj): buildExportAll(obj);`